lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -174,8 +181,18 @@ def add_invoice(post, money, options)
         tx_value[:value] = amount(money)
         tx_value[:currency] = options[:currency] || currency(money)
 
+        tx_tax = {}
+        tx_tax[:value] = options[:tx_tax] || '0'
```

Comment:
@deedeelavinder True, the `tx` for `tx_tax` and `tx_tax_base` is extraneous from the perspective of the transactor, probably worth changing.